### PR TITLE
move route section 'after create a route to the application' option

### PR DESCRIPTION
**Pull request by vikram-raj**
**Fixes**: 
https://issues.redhat.com/browse/ODC-1492

**Analysis / Root cause**: 
The empty routing area looks odd when the `create a route to the application` checkbox is unchecked. Seems to me like it would be better to disable the form fields or at least show a message that routing hasn't been checked.

**Solution Description**: 
move route section after  'after create a route to the application' option

**Screen shots / Gifs for design review**: 
image
image
image
image

**Unit test coverage report**: 
image

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/dev-console/src/components/import/route/RouteSection.tsx [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
 import CreateRoute from './CreateRoute';
 import SecureRoute from './SecureRoute';
 
-interface RouteSectionProps {
-  route: RouteData;
-}
-
-const RouteSection: React.FC<RouteSectionProps> = ({ route }) => {
-  const { t } = useTranslation();
+const RouteSection: React.FC = () => {
```
Comment (sahil143) [inline frontend/packages/dev-console/src/components/import/route/RouteSection.tsx:5]: @vikram-raj I dont think we need this wrapper component anymore, You can directly use CreateRoute, SecureRoute in the AdvancedRouteOption component

File: frontend/packages/dev-console/src/components/import/route/__tests__/AdvancedRouteOptions.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  beforeEach(() => {
+    props = {
+      canCreateRoute: true,
+      resources: Resources.OpenShift,
+    };
+  });
+
+  it('Render AdvancedRoutingOptions', () => {
+    const component = shallow(<AdvancedRouteOptions {...props} />);
+    expect(component.exists()).toBe(true);
```
Comment (sahil143) [inline frontend/packages/dev-console/src/components/import/route/__tests__/AdvancedRouteOptions.spec.tsx:29]: component will always exist. If you want you can check for `isEmptyRender()`

File: frontend/packages/dev-console/src/components/import/route/RouteSection.tsx (hunk 1 above)
Comment (vikram-raj) [reply frontend/packages/dev-console/src/components/import/route/RouteSection.tsx]: Removed this wrapper component.

**Review by sahil143 — APPROVED** [review]
/lgtm

**Pull request merged**